packages/block-library/src/file/save.js

```diff
@@ -20,7 +20,7 @@ export default function save( { attributes } ) {
 					<a
 						href={ textLinkHref }
 						target={ textLinkTarget }
-						rel={ textLinkTarget ? 'noreferrer noopener' : false }
```

Comment:
@adamsilverstein I tested image, buttons and file blocks to see whether a deprecation is needed. It looks like the file block requires a deprecation because the save function is directly modified here.

The other two blocks will keep the existing `rel` values for existing blocks in posts and only use `noopener` for new blocks added.

Reply:
Ok, good catch, I haven't written a deprecation, I will give it a try though.